frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -1,6 +1,6 @@
 import cx from 'classnames'
 
-import { capitalize } from 'lodash'
+import { capitalize, debounce } from 'lodash'
```

Comment:
try to make this file smaller — GitHub is not displaying this file by default because it's too huge

Reply:
refactor PR